src/charts/helpers/export.js

```diff
@@ -116,11 +126,15 @@ function createCanvas(width, height) {
 /**
  * Create Image
  * @param  {string} svgHtml string representation of svg el
+ * @param  {Function} callback function to prepare image for loading
  * @return {object}  TYPE element <img>, src points at svg
  */
-export function createImage(svgHtml) {
+function createImage(svgHtml, callback) {
     let img = new Image();
 
+    if (callback) {
```

Comment:
What do you think about adding the check for the callback to be a function so we can error out with a helpful  message?

Reply:
This is a great suggestion. Will check do now.